packages/dd-trace/test/appsec/index.spec.js

```diff
@@ -645,6 +646,65 @@ describe('AppSec Index', () => {
       })
     })
   })
+
+  describe('Metrics', () => {
+    const appsecNamespace = telemetryMetrics.manager.namespace('appsec')
+    let config
+
+    beforeEach(() => {
+      sinon.restore()
+
+      appsecNamespace.reset()
+
+      config = new Config({
+        appsec: {
+          enabled: true
+        }
+      })
```

Comment:
that's already done line 38 no ?

Reply:
The waf doesn't validate the rules provided in line 38...